### PR TITLE
Add highlighting

### PR DESCRIPTION
**Pull request by Alfus**
Based on pbkit and protobuf.com specifications with some tweaks to make code snippets highlight better.

### CODE REVIEW

File: package.json
Comment (paul-sachs) [inline package.json:1]: Hmm, it's off that the `package-lock.json` file wasn't updated. Is it possible that was missed?

Comment (Alfus) [reply package.json:1]: I couldn't get it to run locally until also updating eslint/parser. Should be fixed now, as long as updating that dep was also ok.

Comment (paul-sachs) [reply package.json:1]: Ah yes, the wonders of npm. I need to spend time on this repo making it less broken, you actually can get it working without updating eslint with `npm install --legacy-peer-deps`

Comment (Alfus) [reply package.json:1]: got it working with the original parser version, thanks!

**Review by paul-sachs — APPROVED** [review]

**Pull request merged**